torchvision/models/detection/rpn.py

```diff
@@ -153,8 +149,8 @@ def grid_anchors(self, grid_sizes, strides):
         return anchors
 
     def cached_grid_anchors(self, grid_sizes, strides):
```

Comment:
why only `strides` should be a tensor, and not `grid_sizes`? Is it because `grid_sizes` is returned by `shape` and in ONNX this becomes `tensor`?

Reply:
This is because of line 164 where I'm changing int() to torch.tensor(... dtype=int64) for strides.
This was not required for grid_sizes
1) I can change grid_sizes to match this type for now.
2) There's a follow up item which is to find why int() and float() result in constants in the graph.